ktor-bom/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
```

Comment:
Can't we avoid using custom plugin? I believe it may lead to serious gradle version limitations. For example, when we will need to upgrade Gradle to support newer metadata, the plugin may be not ready yet for newer versions and  it's release cycles may be too slow.

Reply:
It produce standard artifact like all others: io.ktor:ktor-bom, packaging is pom, all pom metadata handled by `gradle/publish.gradle` just like all others artifacts in the repo

Reply:
Well, this plugin managed by spring team, and already used for bom in kotlinx-coroutines. I assume it pretty stable and doesn't cause issues with new gradle releases. It could be rewrited to pure xml stuff, but I don't think it necessary. Do you insist on removing this plugin?